lib/core/src/lib/ui-header/ui-header.component.md

```diff
@@ -0,0 +1,132 @@
+# UI Header Component
+
+`standalone`, `component`
+
+The UI Header component is a versatile component used to display a header section in applications. It supports various configurations including minimal and extended variants, customizable logo, title, and navigation items.
+
+## Usage
+
+Displaying a minimal header with a logo and navbar:
+
+```html
+<adf-ui-header
+    variant="minimal"
+    headerHeight="64px"
+    color="primary"
+    [logoSrc]="src"
+    [logoAlt]="alt"
+    [logoHeight]="logoHeight"
+    [logoWidth]="logoWidth"
+    [title]="title"
+    [navbarItems]="items"
+>
+</adf-ui-header>
+```
+
+Toolbar actions can be added to the header by placing them inside a `div` element with the `adf-toolbar-actions` class.
+
+```html
+<adf-ui-header
+    variant="minimal"
+    [logoSrc]="src"
+    [title]="title"
+    [navbarItems]="items"
+>
+    <div class="adf-toolbar-actions">
+        <adf-toolbar-divider></adf-toolbar-divider>
+        <adf-button variant="icon" icon="search" (click)="onSearchClick()"></adf-button>
+        <adf-avatar [src]="src"></adf-avatar>
+    </div>
+</adf-ui-header>
+```
+
+![adf-ui-header-extended.png](../../../../../docs/docassets/images/adf-ui-header-minimal.png)
+
+Displaying an extended header with toolbar actions:
+
+```html
+<adf-ui-header variant="extended" [logoSrc]="src" [title]="title" [navbarItems]="items">
+    <div class="adf-toolbar-actions">
+        <adf-button variant="flat">Secondary</adf-button>
+        <adf-button variant="flat" color="accent">Primary</adf-button>
+
+        <adf-toolbar-divider></adf-toolbar-divider>
+
+        <adf-button variant="icon" icon="search"></adf-button>
+        <adf-button variant="icon" [matMenuTriggerFor]="menu" icon="apps"></adf-button>
+        <mat-menu #menu="matMenu">
+            ...
+        </mat-menu>
+        
+        <adf-avatar
+            initials="HU"
+            cursor="pointer"
+            [matMenuTriggerFor]="userMenu">
+        </adf-avatar>
+        <mat-menu #userMenu="matMenu">
+            ...
+        </mat-menu>
+    </div>
+</adf-ui-header>
+```
+
+![adf-ui-header-extended.png](../../../../../docs/docassets/images/adf-ui-header-extended.png)
```

Comment:
maybe worth moving closer to the markdown file, otherwise it builds a dependency on the `docs/docassets` folder

Reply:
moved to `.docs-images/` if that's fine